src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/generate_pipeline_stacks.py

```diff
@@ -30,8 +30,12 @@ def main():
     for template_path in _templates:
         with open(template_path, encoding="utf-8") as template:
             stack_input = json.load(template)
-            app = core.App()
-            PipelineStack(app, stack_input)
+            app = App()
+            PipelineStack(
+                app,
+                stack_input,
+                synthesizer=BootstraplessSynthesizer(),
```

Comment:
Thanks for adding this!

Reply:
To be honest, I could have used the `DefaultStackSynthesizer` with the `generate_bootstrap_version_rule` parameter set to `False`, but since there are no assets in this stack, it makes more sense to use a synthesizer that doesn't require bootstrap in the first place.